Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM ubuntu:16.04 
+USER root
+
+WORKDIR /tests
+
+ARG STAGE_CONFIG="{'zut':'allures'}"
+
+ENV MOZ_HEADLESS="1"
+ENV RUST_BACKTRACE="full"
+
+RUN apt-get -y update && \
+    apt-get install -y software-properties-common &&\
+    apt-get install -y wget unzip && \
+    apt-get install -y libreadline-gplv2-dev libncursesw5-dev libssl-dev libsqlite3-dev tk-dev libgdbm-dev libc6-dev libbz2-dev  && \
```

Comment:
Should we merge the 3 `apt-get install -y` commands (L12-14) into a single command? Not sure we'd see a perf difference either way, so maybe keeping separate is more preferable.